project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.block.inc

```diff
@@ -200,8 +200,8 @@ function _c4m_user_profile_block_user_confirm_email() {
  */
 function _c4m_user_profile_block_user_my_profile_groups() {
   module_load_include('inc', 'c4m_features_homepage', 'plugins/content_types/my_groups/my_groups');
-  $user = menu_get_object('user');
-  $groups = c4m_features_homepage_get_user_groups_grouped_by_membership($user, array('group'));
+  $account = menu_get_object('user');
+  $groups = c4m_features_homepage_get_user_groups_grouped_by_membership($account, array('group'));
```

Comment:
@DavidHernandez 
why not just changing this one in here:
https://github.com/capacity4dev/capacity4more/pull/1155/files#diff-3f72bf9fc8c0ee2206d46d044126106fR204
?